Prevents PHP Warnings on multicurrency_rate.php

PsyCrow-code opened the pull request:
# FIX PHP Warnings on multicurrency_rate.php
Prevents PHP Warnings about undefined variables
$type
$texte
$newcardbutton
and Undefined array key "nbfield"

PsyCrow-code committed:
Prevents PHP Warnings on multicurrency_rate.php

Prevents PHP Warnings about undefined variables
$type
$texte
$newcardbutton
and Undefined array key "nbfield"

eldy commented on htdocs/multicurrency/multicurrency_rate.php:
```diff
@@ -64,6 +64,10 @@
 $sortorder 			= GETPOST('sortorder', 'aZ09comma');
 $page = (GETPOST("page", 'int') ? GETPOST("page", 'int') : 0);
+// Prevents PHP Warnings
+$type = ''; 			// PHP Warning:  Undefined variable $type on line 440
```
Can you remove all comment // PHP Warning ...
This is already provided into the PR title and we can't add such comment at each init of a variable (init should be the standard behaviour so no comment required).

PsyCrow-code committed:
Update multicurrency_rate.php

Avoid PHP Warnings about undefined variables: $type, $texte, $newcardbutton and Undefined array key "nbfield"

PsyCrow-code reviewed (commented):
Unnecessary inline comments removed
